src/lava/lib/dl/slayer/utils/assistant.py

```diff
@@ -258,3 +258,287 @@ def valid(self, input, target):
                 return output
 
             return output, count
+
+
+class DECOLLEAssistant(Assistant):
+    """Assistant that bundles training, validation and testing workflow
+        for DECOLLE models
+
+    Parameters
+    ----------
+    net : torch.nn.Module
+        network to train.
+    error : object or lambda
+        an error object or a lambda function that evaluates error.
+        It is expected to take ``(output, target)`` | ``(output, label)``
+        as it's argument and return a scalar value.
+    optimizer : torch optimizer
+        the learning optimizer.
+    stats : slayer.utils.stats
+        learning stats logger. If None, stats will not be logged.
+        Defaults to None.
+    classifier : slayer.classifier or lambda
+        classifier object or lambda function that takes output and
+        returns the network prediction. None means regression mode.
+        Classification steps are bypassed.
+        Defaults to None.
+    count_log : bool
+        flag to enable count log. Defaults to False.
+    lam : float
+        lagrangian to merge network layer based loss.
+        None means no such additional loss.
+        If not None, net is expected to return the accumulated loss as second
+        argument. It is intended to be used with layer wise sparsity loss.
+        Defaults to None.
+
+    Attributes
+    ----------
+    net
+    error
+    optimizer
+    stats
+    classifier
+    count_log
+    lam
+    device : torch.device or None
+        the main device memory where network is placed. It is not at start and
```

Comment:
It is None at start?

Reply:
It is, it was a typo in the Assistant docstring, I have fixed both